plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/model/AltibaseReplicationItem.java

```diff
@@ -0,0 +1,75 @@
+package org.jkiss.dbeaver.ext.altibase.model;
```

Comment:
```suggestion
/*
 * DBeaver - Universal Database Manager
 * Copyright (C) 2010-2023 DBeaver Corp and others
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package org.jkiss.dbeaver.ext.altibase.model;
```

Reply:
Another missing license header. Do you have any tool to check license header text in source file?

Reply:
I found https://github.com/lluissm/license-header-checker and looks working well. I'll use it before committing. Anyway, thanks for your reply.